Sprint-2/implement/cases.js

```diff
@@ -13,3 +13,20 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+
+
+function toUpperSnakeCase(inputString) {
+    // Convert the string to uppercase and replace spaces with underscores
+    return inputString.toUpperCase().replace(/ /gi, "_");
```

Comment:
Can you explain why you included the `i` flag in your regular expression `/ /gi`?